picard/ui/util.py

```diff
@@ -27,9 +27,7 @@
     IS_MACOS,
     IS_WIN,
 )
-from picard.util import (
-    find_existing_path,
-)
+from picard.util import find_existing_path
```

Comment:
I'm not fond of those back and forth changes. They will happen often, a solution would be to always wrap, or to use force single line, neither are great. Prolly have to live with it.

Reply:
I don't think it's so bad. It only happens if you change the imports, and it is a very clear change anyway. You can't fully prevent formatting changes when you touch code.